irrep/utility.py

```diff
@@ -296,7 +296,7 @@ def orthogonalize(A, warning_threshold=np.inf, error_threshold=np.inf , verbosit
     if np.any(np.abs(s - 1) > error_threshold):
         raise ValueError(f"Matrix is not orthogonal \n {A} \n {debug_msg}")
     elif np.any(np.abs(s - 1) > warning_threshold):
-        log_message(f"Warning: Matrix is not orthogonal \n {A} \n {debug_msg}", verbosity, 1)
```

Comment:
why this change? The function `log_message` takes as `verbosity` and `level` as positional args. We need to provide both when calling it.